src/main/resources/openapi/event.yaml

```diff
@@ -860,6 +892,7 @@ components:
         MAPPING_IN_USE,
         TOO_MANY_RESULTS,
         EVENT_ID_NOT_FOUND,
+        ALREADY_OBFUSCATED
```

Comment:
This error doesn't appear to be used and I don't see any use for it. If the event is already obfuscated then we just return 200.

Can it be removed?

Reply:
Agreed updated